dnsmasq/files/dnsmasq.init

```diff
@@ -455,6 +460,8 @@ dhcp_add() {
 		DNS_SERVERS="$DNS_SERVERS $dnsserver"
 	}
 
+	dnsmasq_device_list="${dnsmasq_device_list} $ifname"
+	dnsmasq_interface_list="${dnsmasq_interface_list} $net"
```

Comment:
- dnsmasq_device_list is used to list network devices used by dnsmasq to set dhcp-ignore on all other device like loopback, tun0, xtun0...
```
dhcp-host=a4:5e:60:c4:c8:89,set:ovh1,192.168.0.25,astreinte

dhcp-range=set:lan,192.168.0.100,192.168.0.249,255.255.255.0,12h
dhcp-option=lan,43,01:04:A7:72:F2:2A
dhcp-option=lan,option:router,192.168.0.20
dhcp-option=lan,option:dns-server,192.168.0.20

dhcp-range=ovh1,192.168.0.1,static,255.255.255.0,12h
dhcp-option=ovh1,option:router,192.168.0.251
dhcp-option=ovh1,option:dns-server,8.8.8.8

no-dhcp-interface=eth0
no-dhcp-interface=if0
no-dhcp-interface=if1
no-dhcp-interface=lo
no-dhcp-interface=ovh2
no-dhcp-interface=tun0
```

- dnsmasq_interface_list is used to list interfaces where a dhcp is configured, that allow to setup a static dhcp when a host is tagged with an interface that is not configured.

```
config dhcp 'lan'
	option interface 'lan'
	option start '100'
	option limit '150'
	option leasetime '12h'
	list dhcp_option '43,01:04:A7:72:F2:2A'

config dhcp 'wan'
	option interface 'wan'
	option ignore '1'

config host
	option name 'astreinte'
	option mac 'a4:5e:60:c4:c8:89'
	option ip '192.168.0.25'
	option tag 'ovh1'
```